client/src/pages/course/admin/interviews.tsx

```diff
@@ -46,20 +48,39 @@ function Page() {
     }
   };
 
+  const createInterviews = async () => {
+    if (selected) {
+      await courseService.createInterviewDistribution(+selected);
```

Comment:
```suggestion
      await courseService.createInterviewDistribution(Number(selected));
```

it's better to write conversions in an explicit way

Reply:
@valerydluski mentioned today that on courses with a large number of students, creating interview pairs can take a significant amount of time. Without showing a loading screen, users won’t understand that something is in progress and might click the button again. This leads to duplicate pairs being created, which later have to be cleaned up manually in the DB :(

So, we need to:

1) Add a loading screen that stays visible for the entire operation.
2) Leave clear comments for future developers explaining why this is necessary.
3) If possible, refactor the createInterviewDistribution logic to make sure the process can't be triggered multiple times at once.

P.S. Big thanks to @valerydluski Megamind, for his attention and alertness 😎

Reply:
Number(selected) fixed.